pkg/webhooks/policyvalidation.go

```diff
@@ -47,7 +48,9 @@ func (ws *WebhookServer) policyValidation(request *v1beta1.AdmissionRequest) *v1
 			},
 		}
 	}
-
+	if len(response.Warnings) != 0 {
```

Comment:
we have to check the nil response as well 
```suggestion
	if  response != nil && len(response.Warnings) != 0 { 
```

Reply:
Ah, good catch, thanks :)